frontend/src/components/operation_badges_modal/index.tsx

```diff
@@ -0,0 +1,69 @@
+// Copyright 2022, Yahoo Inc.
+// Licensed under the terms of the MIT. See LICENSE file in project root for terms.
+
+import * as React from 'react'
+import classnames from 'classnames/bind'
+import { EvidenceCount, OperationStatus, operationStatusToLabel, TopContrib } from 'src/global_types'
+import Modal from '../modal'
+
+const cx = classnames.bind(require('./stylesheet'))
+
+export default (props: {
+  onRequestClose: () => void,
+  topContribs?: Array<TopContrib>,
```

Comment:
I need to take a closer look, but in general I think it's better to require an array, but when not present, have it act the same as `[]`.  It looks like that would make the code simpler in one place, at least.

Reply:
ohhh good call - I'll take a look at that

Reply:
I removed the optional ? (and a few others that were scattered around that are provided by the backend), but didn't see where it would make the code simpler - where is that?

Reply:
I think I overstated simplified. I guess I consider `a.b` simpler than `a.?b`, which I suppose _is_ but not appreciably so.

Reply:
oh haha no worries